src/redux/features/votes/votesAsyncThunks.ts

```diff
@@ -17,10 +17,11 @@ export const requestProjectVotesGet = createAsyncThunk('votes/get', async (proje
 
 export const requestProjectVotesUpdate = createAsyncThunk(
   'votes/update',
-  async (request: { projectId: number; payload: UpsertProjectVotesRequestPayload }, { rejectWithValue }) => {
+  async (request: { projectId: number; payload: UpsertProjectVotesRequestPayload }, { dispatch, rejectWithValue }) => {
     const response: Response = await VotesService.setProjectVotes(request.projectId, request.payload);
 
     if (response.requestSucceeded) {
+      dispatch(requestProjectVotesGet(request.projectId));
```

Comment:
Thanks for this discussion. This is fine.
Thoughts on API requests being more granular and have the UX handle the intent of refetching data? The current approach definitely seems simpler but was curious about separation of concern.